src/src/urls.py

```diff
@@ -43,6 +45,20 @@
 router.register(r'convert', views.ConvertViewSet)
 router.register(r'compare', views.CompareViewSet)
 
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Blog API",
+        default_version="v1",
+        description="A sample API for learning DRF",
+        terms_of_service="https://www.google.com/policies/terms/",
+        contact=openapi.Contact(email="hello@example.com"),
+        license=openapi.License(name="BSD License"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
+
```

Comment:
The `redoc/` and `swagger/` endpoints are not showing any parameters and response for drf apis, 
<img width="1427" alt="Screenshot 2022-03-13 at 7 02 26 PM" src="https://user-images.githubusercontent.com/17159160/158061923-baa542f6-fd8b-4593-b77c-c254c37fe546.png">

The values are coming up empty. How will drf-yasg pick up parameters information from the serializer defined in views.py file?